File: easy/objective-c/onboarding.m
#import <Foundation/Foundation.h>

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        while (true) {
            char enemy1[50], enemy2[50];
            int dist1, dist2;

            scanf("%s %d", enemy1, &dist1);
            scanf("%s %d", enemy2, &dist2);

            if (dist1 < dist2) {
                printf("%s\n", enemy1);
            } else {
                printf("%s\n", enemy2);
            }
        }
    }
    return 0;
}

File: easy/F#/onboarding.fs
open System

[<EntryPoint>]
let main argv =
    while true do
        let enemy1 = Console.ReadLine()
        let dist1 = Int32.Parse(Console.ReadLine())
        let enemy2 = Console.ReadLine()
        let dist2 = Int32.Parse(Console.ReadLine())

        if dist1 < dist2 then
            printfn "%s" enemy1
        else
            printfn "%s" enemy2

    0
